src/collision/shapes/chain.rs

```diff
@@ -55,9 +55,9 @@ impl ChainShape {
 
     pub fn prev_vertex(&self) -> Option<Vec2> {
         unsafe {
-            let mut v = mem::uninitialized();
-            if ffi::ChainShape_get_prev_vertex(self.ptr(), &mut v) {
-                Some(v)
+            let mut v = mem::MaybeUninit::uninit();
+            if ffi::ChainShape_get_prev_vertex(self.ptr(), v.get_mut()) {
```

Comment:
maybe you can use `&mut *v.as_mut_ptr()` for these cases to only use stable features?